### DISCUSSION TITLE
Acabo de crear mi primera calculadora sin ayuda

### DISCUSSION THREAD

**Discussion by Kevin-Miguel** [community/community#123700] [Programming Help]
import time

while True:
    try:
        print("¿Que operacion quieres hacer?")
        print("1 |sumar")
        print("2 |restar")
        print("3 |multiplicar")
        print("4 |dividir")
        opcion = int(input("Elija una opcion de las 4 de arriba (Ingresa el numero): "))
        primer_numero = int(input("Ingresa el primer numero: "))
        segundo_numero = int(input("ingresa el segundo numero: "))
        
        if  opcion == 1:
            resultado = primer_numero + segundo_numero
            
        elif opcion == 2:
            resultado = primer_numero - segundo_numero
            
        elif opcion == 3:
            resultado = primer_numero * segundo_numero
            
        elif  opcion == 4:
            resultado = primer_numero / segundo_numero
            
        else:
            print("Opcion no valida")
            exit()
            

        print(f"Tu resultado es: {resultado}")
        time.sleep(3)
    
    except ValueError:
        print("Ingresa un numero valido")

**Comment by LiteBrite82** [comment]
Hi @Kevin-Miguel , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
